Wikipedia/Localizations/br.lproj/Localizable.strings

```diff
@@ -394,7 +394,7 @@
 "home-featured-article-heading" = "Pennad eus ar c'hentañ evit $1";
 "home-main-page-heading" = "Hiziv e Wikipedia";
 "home-potd-heading" = "Skeudenn an deiz evit $1";
-"home-last-update-label" = "Hizivadenn diwezhañ: $1";
+"home-last-update-label" = "Hizivadenn diwezhañ: %1$";
 "home-last-update-never-label" = "Hizivadenn diwezhañ : Morse";
```

Comment:
@joewalsh 

I think I know what's happening here. Looking at the [revision history](https://translatewiki.net/w/i.php?title=Wikimedia:Wikipedia-ios-home-last-update-label/br&action=history) of this string it appears our changes (i.e. when we correct the string to use "$1" vs "%1$") are no longer getting automatically back-propagated. It was my understanding that formerly this happened automatically. 

Perhaps some of the recent TWN changes broke this? I guess it's not necessarily "broken" so long as we understand we'll need to go to translatewiki.net and edit such strings in the future so the same un-updated strings doesn't keep coming down each time we sync...

Thoughts?